<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QPushButton" name="button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="addButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#FF8C00;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: #FF4500;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="multiplyButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#FF8C00;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: #FF4500;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="subtractButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#FF8C00;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: #FF4500;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="divideButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#FF8C00;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: #FF4500;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="clearButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="signChangeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#C0C0C0;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: A9A9A9;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="memoryAddButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#FF8C00;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: #FF4500;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="memoryClearButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#FF8C00;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: #FF4500;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="memoryGetButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#FF8C00;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: #FF4500;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="equalButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color:#FF8C00;
  border: 1px solid gray;
  padding: 5px;
 color: black
}

QPushButton:pressed{
  background-color: #FF4500;
  border: 1px solid gray;
  padding: 5px;
 color: black;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
  background-color: #C0C0C0;
  border: 1px solid gray;
  padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
